package.json

```diff
@@ -25,11 +25,11 @@
   },
   "homepage": "https://github.com/devongovett/browserify-istanbul",
   "dependencies": {
-    "istanbul": "~0.3.6",
     "minimatch": "^2.0.0",
     "through": "^2.3.4"
   },
   "devDependencies": {
+    "istanbul": "^0.4.2",
```

Comment:
Since instrumentation by itself is of little use (*this is what this module does*) and requires to run reporting after that (*this module doesn't provide*), then having instrumentation completed with istanbul version `A` and reporting ran using istanbul version `B` will be much worse, than just failing at instrumentation step. 

Best way would be to use `peerDependency` but at the moment it's broken in npm, so we're stuck with next best thing – omitting that dependency altogether.